### DISCUSSION TITLE
useAtomDevtools doesn't show atoms in Redux DevTools

### DISCUSSION THREAD

**Discussion by phuctm97** [jotaijs/jotai-devtools#128] [Q&A] [answered]
I'm trying to use useAtomDevtools to debug atoms in Redux DevTools but it doesn't seem to work.

I use only the `useAtomDevtools` from `jotai-devtools`, no UI `<DevTools />` or anything others, I'm not sure if there is any requirement for it to work, the docs doesn't mention any requirements.

**Comment by arjunvegda** [comment, marked as answer]
Thanks for reaching out @phuctm97. 

The main entry point relies on `@emotion/react` so you'd need to install that as a dependency (see [docs](https://github.com/jotaijs/jotai-devtools?tab=readme-ov-file#-setup)).  If you don't plan on using `<DevTools/>` UI then I'd recommend using `jotai-devtools/utils` entry point to import the hook. 

Here is a working CSB - https://codesandbox.io/p/sandbox/discussion-127-r7gw5s 🙌 

<img width="1433" alt="image" src="https://github.com/jotaijs/jotai-devtools/assets/14841132/2f995465-8eea-4ec7-b655-375a86454dd5">

**Reply by phuctm97** [reply]
Thank you @arjunvegda! IT works perfectly without requiring `@emotion/react` now!
